db/seeds.rb

```diff
@@ -41,14 +43,31 @@
   'Townhouse Apts'       => 30
 }
 
+# if this array changes then the logic below has to also change
+directions = ['North', 'South']
+
 stops.each do |route_number, stop_names|
-  stop_names.each do |stop_name|
-    # Anytime in the last two months
-    Timecop.freeze rand(86_400).minutes.ago do
-      stop = BusStop.find_or_initialize_by name: stop_name
-      stop.hastus_id = hastus_ids.fetch(stop_name)
-      stop.routes << routes.fetch(route_number)
-      stop.save!
+  directions.each do |direction|
+    if direction == 'North'
+      sequence = 0
+    else
+      sequence = stop_names.length
+    end
+    stop_names.each do |stop_name|
+      if direction == 'North'
+        sequence += 1
+      else
+        sequence -= 1
+      end
```

Comment:
That's an interesting algorithm. I think, like in the rake task, it would be simpler to construct the lists of stops first, then give them sequence numbers according to where they are in the list (`each.with_index(1)`, etc.).

This is *clever*, but not particularly intuitive, in my view.

Reply:
Dammit. Okay.